SPECS/intel-desktop-virtualization-k3s/intel-desktop-virtualization-k3s.spec

```diff
@@ -0,0 +1,54 @@
+Name:           intel-desktop-virtualization-k3s
+Version:        0.1
+Release:        1%{?dist}
+Vendor:         Intel Corporation
+Distribution:   Edge Microvisor Toolkit
+URL:            https://github.com/open-edge-platform/edge-desktop-virtualization
+Summary:        Provides Kubevirt (enabled with GTK libarary support and Intel SR-IOV patched QEMU in Virt-Launcher) and IDV Device Plugin for enabling support of local GTK display using pre-built container tar files
+
+License:        Apache-2.0
+Source0:        https://github.com/open-edge-platform/edge-desktop-virtualization/releases/download/pre-release-v0.1/intel-idv-kubevirt-v0.1.tar.gz
+Source1:        https://github.com/open-edge-platform/edge-desktop-virtualization/releases/download/pre-release-v0.1/intel-idv-device-plugin-v0.1.tar.gz
+BuildArch:      x86_64
+Requires:       k3s
+
+%description
+Provides Kubevirt (enabled with GTK libarary support and Intel SR-IOV patched QEMU in Virt-Launcher) and IDV Device Plugin for enabling support of local GTK display using pre-built container tar files
+
+%prep
+tar -xzf %{SOURCE0} -C .
+tar -xzf %{SOURCE1} -C .
+
+%build
+
+%install
+mkdir -p %{buildroot}%{_sharedstatedir}/rancher/k3s/agent/images/
+cp *.tar %{buildroot}%{_sharedstatedir}/rancher/k3s/agent/images/
+
+mkdir -p %{buildroot}%{_sharedstatedir}/rancher/k3s/server/manifests/
+cp *.yaml %{buildroot}%{_sharedstatedir}/rancher/k3s/server/manifests/
+
+%files
+%{_sharedstatedir}/rancher/k3s/agent/images/virt-api.tar
+%{_sharedstatedir}/rancher/k3s/agent/images/virt-controller.tar
+%{_sharedstatedir}/rancher/k3s/agent/images/virt-handler.tar
+%{_sharedstatedir}/rancher/k3s/agent/images/virt-launcher.tar
+%{_sharedstatedir}/rancher/k3s/agent/images/virt-operator.tar
+%{_sharedstatedir}/rancher/k3s/agent/images/busybox.tar
+%{_sharedstatedir}/rancher/k3s/agent/images/device-plugin.tar
+%{_sharedstatedir}/rancher/k3s/agent/images/sidecar-shim.tar
+%{_sharedstatedir}/rancher/k3s/server/manifests/device-plugin.yaml
+%{_sharedstatedir}/rancher/k3s/server/manifests/kubevirt-cr.yaml
+%{_sharedstatedir}/rancher/k3s/server/manifests/kubevirt-operator.yaml
+
+
+%post
+
+%changelog
+* Fri Jun 20 2025 Dhanya A <dhanya.a@intel.com> - v0.1-2
```

Comment:
since this is new specs the "release" should be 1 and all the changelog can be merge into single entries for version "0.1-1"